suite-common/message-system/config/config.v1.json

```diff
@@ -147,6 +147,64 @@
                 }
             }
         },
+        {
+            "conditions": [
+                {
+                    "settings": [
+                        {
+                            "matic": true
+                        }
+                    ],
+                    "environment": {
+                        "desktop": "<=24.9.2",
+                        "mobile": "!",
```

Comment:
we will be releasing POL for mobile 24.11.1 next week. I guess that this message should be displayed there as well.

Reply:
But you will be already using the correct symbol (POL) and new backends, right? This message is aimed for users on older Suites that are currently connected to old BE with MATIC as a symbol (so before migration).